contents/performance/WebPerformance.md

```diff
@@ -0,0 +1,80 @@
+<frontmatter>
+  title: Web Performance
+  footer: footer.md
+  head: head.md
+  siteNav: mainNav.md
+  pageNav: 3
+</frontmatter>
+
+{{ navbar | safe }}
+
+<div class="website-content">
+
+# Web Performance
+
+Author: [Marvin Chin](https://github.com/marvinchin)
+
+## What is Web Performance
+
+Web Performance is a broad term that refers to how performant a web application *feels* to its users. This includes many aspects such as how long the site takes to load, how quickly the site becomes interactive, and how responsive it feels when the user interacts with the site.
+
+As web applications grow more complex, it becomes increasingly important for web developers to be aware of the factors that affect performance, and consider how performance can impact the user experience.
+
+## Why Web Performance Matters
+
+The performance of your web application has a direct impact on its ability to attract and retain users: 
+- Sites which take more than 3 seconds to load are [more likely to be abandoned by users](https://developer.akamai.com/blog/2016/09/14/mobile-load-time-user-abandonment).
```

Comment:
Not clear why that part of the sentence is hyperlinked but not the rest. Do one of these two?
> [Sites which take more than 3 seconds to load are more likely to be abandoned by users]()

>Sites which take more than 3 seconds to load are more likely to be abandoned by users [[source]()]

This point applies to other such links too.